crates/optimism/txpool/src/interop.rs

```diff
@@ -6,7 +6,7 @@ pub trait MaybeInteropTransaction {
     fn set_interop(&self, interop: TransactionInterop);
 
     /// Get attached [`TransactionInterop`] if any.
-    fn interop(&self) -> Option<TransactionInterop>;
+    fn interop(&self) -> TransactionInterop;
```

Comment:
this should still be an option, this way we don't need to check validity at the callsite everytime